pkg/apis/openebs/v1alpha1/jivavolume_types.go

```diff
@@ -28,6 +28,7 @@ type ISCSISpec struct {
 }
 
 type MountInfo struct {
+	TargetPath string `json:"targetPath"`
 	Path       string `json:"mountPath"`
```

Comment:
It is a bit confusing, can we rename the above variables as TargetPath and StagingTargetPath.
mountPath might not be a valid name for block devices.